pg_python/pg_test.py

```diff
@@ -179,9 +179,11 @@ def test_update_multicol_2testcond(self):
 
 class TestCompany(unittest.TestCase):
     def test_company_formatter(self):
-        company_name = "PYROTECH ELECTRONICS (P) Ltd."
-        formated_name = pg_company.find_proper_name(company_name)
-        self.assertEqual(formated_name, "Pyrotech Electronics Private Limited")
+        self.assertEqual(find_proper_name("ENDRESS + HAUSER (I) (PVT). 'LTD"), "Endress + Hauser (India) Private Limited")
+        self.assertEqual(find_proper_name("PYROTECH ELECTRONICS (P) Ltd."), "Pyrotech Electronics Private Limited")
+        self.assertEqual(find_proper_name("LPS Bossard (pv$ limite$)"), "LPS Bossard (Private Limited)")
```

Comment:
LPS Bossard (Private Limited) should be LPS Bossard Private Limited https://www.indiamart.com/lps-bossardltd/aboutus.html.

Reply:
@ingloriousb updated the dict pair & test case result https://github.com/ingloriousb/pg_python/pull/4/commits/bed198d742306c8174ae0e0ea87f3aebc24ebbce

Reply:
@ingloriousb updated the dict pair & test case result https://github.com/ingloriousb/pg_python/pull/4/commits/bed198d742306c8174ae0e0ea87f3aebc24ebbce